packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -99,7 +110,29 @@ const RiskManagementItem = observer(
                 setIsEnabled(false);
             }
             updateLimitOrder();
+            setIsSheetOpen(false);
         };
+
+        const min_value = type && (validation_params?.[type]?.min ?? 0);
+        const max_value = type && (validation_params?.[type]?.max ?? 0);
+
+        const error_message = (() => {
+            const field_label = type === 'take_profit' ? localize('take profit') : localize('stop loss');
+            if (stepperValue && min_value && Number(stepperValue) < min_value) {
```

Comment:
Just for my understanding: you are combining FE and BE validation?
Like to validate on BE first (send a new value (onChange function) -> receive or not receive validation_error_message -> do FE validation -> show FE error (if it is), if no FE, then show BE error?

Reply:
There's no BE validation, it's all just FE

Reply:
I'm not sending anything to BE unless save is pressed and successful

Reply:
Thank you for explanation